js/common/modules/@arangodb/aql/explainer.js

```diff
@@ -2574,6 +2575,31 @@ function debug(query, bindVars, options) {
     result.collections[c.name] = c;
   });
 
+  let used_analyzers = new Set();
+  Object.values(result.views).forEach(v => {
+    let links = v["properties"]["links"];
+    Object.values(links).forEach(c => {
+      let curr_analyzers = c["analyzers"];
+      curr_analyzers.forEach(a => {
+        used_analyzers.add(a);
+      });
+    });
+  });
+
+  let analyzers = require("@arangodb/analyzers");
+  let all_analyzers = analyzers.toArray();
+  used_analyzers.forEach(used_a => {
+    let re = new RegExp(String.raw`^_system::${used_a}$`, "g");
+    let found = all_analyzers.find((a) => a.name().match(re));
+    if(found) {
+      // it means that it is not a system analyzer
```

Comment:
Isn't the current behavior of the code the opposite of what the code comment says?
I guess if `found` is true, it _is_ a system analyzer and not "not a system analyzer".